src/main/java/com/minecolonies/core/colony/managers/TravellingManager.java

```diff
@@ -0,0 +1,216 @@
+package com.minecolonies.core.colony.managers;
+
+import com.google.common.collect.Maps;
+import com.minecolonies.api.colony.ICitizenData;
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.managers.interfaces.ITravellingManager;
+import com.minecolonies.api.entity.citizen.AbstractEntityCitizen;
+import com.minecolonies.api.util.EntityUtils;
+import com.minecolonies.api.util.Log;
+import com.minecolonies.api.util.constant.ColonyConstants;
+import com.minecolonies.api.util.constant.NbtTagConstants;
+import com.minecolonies.core.util.TeleportHelper;
+import net.minecraft.core.BlockPos;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.ListTag;
+import net.minecraft.nbt.NbtUtils;
+import net.minecraft.nbt.Tag;
+import net.minecraftforge.common.util.INBTSerializable;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class TravellingManager implements ITravellingManager, INBTSerializable<CompoundTag>
```

Comment:
docs are missing in this file

Reply:
We have several other managers without docs.  This is not a particular difficult implementation, its API interface has the docs. Is this still needed here?

Reply:
ah you dont need to put doc on the override methods, but the general class desc would be nice for its purpose and on the methods that arent part of the interface